WindowsServerDocs/storage/folder-redirection/troubleshoot-user-profiles-events.md

```diff
@@ -65,8 +65,15 @@ Here's how to create and decode a trace for the User Profile Service:
 2. From an elevated command prompt enter the following commands, where *\<Path\>* is the path to a local folder that you have previously created, for example C:\\logs:
```

Comment:
Even though this line is outside your PR changes, may I suggest a comma adjustment here, for readability? (We try to add improvements whenever possible.)

```suggestion
2. From an elevated command prompt, enter the following commands where *\<Path\>* is the path to a local folder that you have previously created, for example C:\\logs:
```